Mathlib/MeasureTheory/Group/GeometryOfNumbers.lean

```diff
@@ -83,4 +83,56 @@ theorem exists_ne_zero_mem_lattice_of_measure_mul_two_pow_lt_measure [NormedAddC
   refine' h_conv hw (h_symm _ hv) _ _ _ <;> norm_num
 #align measure_theory.exists_ne_zero_mem_lattice_of_measure_mul_two_pow_lt_measure MeasureTheory.exists_ne_zero_mem_lattice_of_measure_mul_two_pow_lt_measure
 
+/-- The **Minkowski Convex Body Theorem for compact domain**. If `s` is a convex compact symmetric
+domain of `E` whose volume is large enough compared to the covolume of a lattice `L` of `E`, then it
+contains a non-zero lattice point of `L`. Compared to
+`exists_ne_zero_mem_lattice_of_measure_mul_two_pow_lt_measure`, this version requires in addition
+that `s` is compact and `L` is discrete but provides a weaker inequality rather than a strict
+inequality. -/
+theorem exists_ne_zero_mem_lattice_of_measure_mul_two_pow_le_measure [NormedAddCommGroup E]
+    [NormedSpace ℝ E] [BorelSpace E] [FiniteDimensional ℝ E] [Nontrivial E] [IsAddHaarMeasure μ]
+    {L : AddSubgroup E} [Countable L] [DiscreteTopology L] (fund : IsAddFundamentalDomain L F μ)
+    (h_symm : ∀ x ∈ s, -x ∈ s) (h_conv : Convex ℝ s) (h_cpt : IsCompact s) (h_mes : μ s ≠ 0)
+    (h : μ F * 2 ^ finrank ℝ E ≤ μ s) :
+    ∃ (x : _) (_ : x ≠ 0), ((x : L) : E) ∈ s := by
+  have h_nemp : s.Nonempty := nonempty_of_measure_ne_zero h_mes
+  let u : ℕ → ℝ≥0 := fun n => 1 / ((n:ℝ≥0) + 1)
```

Comment:
Often, instead of using an explicit sequence tending to zero, it is more efficient to use a lemma like `exists_seq_strictAnti_tendsto` which gives you some sequence with this property.

Reply:
That is indeed very convenient. In this case, that means I don't need the following lemma anymore: 
```lean
theorem NNReal.tendsto_const_div_add_atTop_nhds_0_nat (C : ℝ≥0) :
     Tendsto (fun n : ℕ => C / ((n:ℝ≥0) + 1)) atTop (𝓝 0)
```
Should I remove it from the PR since I am not really convinced that it is really useful in general?

Reply:
Yes, I would remove it.

Reply:
it's gone.